201-redis-premium-persistence/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,62 @@
+﻿{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {},
+    "variables": {
+        "storageAccountName": "[concat('str1',uniqueString(resourceGroup().id))]",
+        "storageAccountName2": "[concat('str2',uniqueString(resourceGroup().id))]"     
+    },
+    "resources": [{
+            "type": "Microsoft.Storage/storageAccounts",
+            "name": "[variables('storageAccountName')]",
+            "apiVersion": "2016-01-01",
+            "location": "West US",
+            "comments": "Storage account used to store VM disks",

```

Comment:
this looks like an old comment, we should update to show the purpose of the account...